pkgs/development/compilers/dale/default.nix

```diff
@@ -21,9 +21,8 @@ in stdenv.mkDerivation {
     sha256 = "0v4ajrzrqvf279kd7wsd9flrpsav57lzxlwwimk9vnfwh7xpzf9v";
   };
 
-  nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ cmake libffi llvm_6 ]
-             ++ stdenv.lib.optional doCheck perl;
+  nativeBuildInputs = [ cmake pkg-config ];
+  buildInputs = [ libffi llvm_6 ] ++ stdenv.lib.optional doCheck perl;
```

Comment:
doCheck sounds like this should be checkInputs but I am not sure.

Reply:
Fixed.